### ISSUE TITLE
WiFi USB is recognized, but the app doesn't show any available network

### ISSUE THREAD

**Issue by BenjaminStonawski** [chris1111/Wireless-USB-Big-Sur-Adapter#334]
Hello there!
I installed this utility on my Early 2015 MacBook Pro running macOS Ventura. I'm using a TP-Link WN823Nv2 WiFi USB adapter. After installing the app, the utility recognizes the adapter but doesn't show any available network (shown in the screenshots below). 
I also tried to manually enter the WiFi network and connect via WPS, but none worked either.
What could be the issue? 
Thank you for your answer in advance!
![Képernyőfotó 2024-09-23 - 0 59 14](https://github.com/user-attachments/assets/6393dd90-3760-4303-b50f-dbceaa4197be)
<img width="539" alt="Képernyőfotó 2024-09-23 - 0 59 45" src="https://github.com/user-attachments/assets/9adec39c-fa3e-4c11-ad31-c825ba33f768">

**Comment by chris1111** [comment]
Try turn Off and On?
Also check your router

**Comment by BenjaminStonawski** [comment]
Yes, I tried to restart the Mac several times and tried different USB ports also. There are more WiFi routers in the area (2-3) but none of them is showing up.

**Comment by chris1111** [comment]
First time seeing this you are certainly using OCLP?

**Comment by BenjaminStonawski** [comment]
Yes, I'm using OpenCore Patcher. Maybe should I try to reinstall root patches?

**Comment by chris1111** [comment]
Uninstall this utility 
Reboot Mac OS then try the one dedicated for OpenCore users

**Comment by BenjaminStonawski** [comment]
I tried that but same error occurred.
I even tried to reinstall root patches and update OpenCore but didn't fix the issue either.

**Comment by chris1111** [comment]
Check your config.plist, some thing wrong probably enter in conflicts with the utility because I have no issue same adapter from 10.15 to macOS 15

**Comment by BenjaminStonawski** [comment]
What could conflict? Are there any specific boot-args that need to be added?

**Comment by BenjaminStonawski** [comment]
Does SIP need to be fully disabled? Because after entering "csrutil disable" Terminal app says "SIP status: unknown".
